xml-parsing-example/flb.conf

```diff
@@ -0,0 +1,20 @@
+[SERVICE]
+    flush      0.1
+    grace      2
+    log_level  info
+
+[INPUT]
+    name tail
+    read_from_head true
+    exit_on_eof true
```

Comment:
Be aware this doesn't flush so your output will be different between runs. Might not be an issue, but it was for me when diffing it!

Reply:
Do you mean `exit_on_eof` ? I used it to make fluent-bit exit when it is done processing the example file. What do you suggest?

Reply:
Yeah, currently there is no option to guarantee all records are flushed. Depends if you care, it's fine most of the time